bin/setwinsize/setwinsize.c

```diff
@@ -0,0 +1,51 @@
+#include <stdlib.h>
+#include <string.h>
+#include <unistd.h>
+#include <stdio.h>
+#include <termios.h>
+#include <term.h>
+#include <sys/fcntl.h>
+#include <sys/ioctl.h>
+
+int main(void) {
+  int err = 1;
+  struct termios old, t;
+  FILE *termfile;
+  if ((termfile = fopen("/dev/tty", "r+")) == NULL)
+    return err;
+  int term_fd = fileno(termfile);
+  if (setupterm(NULL, term_fd, NULL))
+    return err;
+  setbuf(termfile, NULL);
+  /* Check if we can move the cursor and save/restore cursor position. */
+  if (cursor_address == NULL || save_cursor == NULL || restore_cursor == NULL)
+    return err;
+  if (tcgetattr(term_fd, &old))
+    return err;
+  t = old;
+  cfmakeraw(&t);
+  if (tcsetattr(term_fd, TCSAFLUSH, &t))
+    return err;
+  /* Save current cursor position. */
+  fputs(save_cursor, termfile);
+  /* Move the cursor to the bottom right corner of the screen. */
+  const char *buf = tiparm(cursor_address, 1000, 1000);
```

Comment:
What does 1000 mean?

Reply:
They are the row and column numbers that we're moving the cursor to. I have added more explanatory comments.